purchases/src/main/kotlin/com/revenuecat/purchases/PostReceiptHelper.kt

```diff
@@ -93,6 +95,7 @@ internal class PostReceiptHelper(
             presentedOfferingContext = purchase.presentedOfferingContext,
```

Comment:
Hmm I'm thinking that the presented offering context we post to the backend will always be for the "base" package/product and won't include the add-ons if they were obtained from a different placement or something... I think it might be an acceptable compromise for now but I think we should be aware of this limitation.

Reply:
I think this is ok, we probably only care about the base product in that scenario, but good callout